tests/testthat/test-clean-names.R

```diff
@@ -186,6 +186,35 @@ test_that("do not create duplicates (fix #251)", {
   )
 })
 
+test_that("labels are created in default and sf methods (feature request #563)", {
+  dat_df <- dplyr::tibble(`a a` = c(11, 22), `b b` = c(2, 3))
+  dat_df_clean_labels <- clean_names(dat_df, set_labels = TRUE)
+  dat_df_clean <- clean_names(dat_df)
+  
+  dat_sf <- dat_df
+  dat_sf$x <- c(1,2)
+  dat_sf$y = c(1,2) 
```

Comment:
Please use left arrow (`<-`) assignment instead of equal sign assignment to match the style of the rest of the package.

Reply:
Done